release_dashboard/test/widgets/conductor_status_test.dart

```diff
@@ -81,7 +78,7 @@ void main() {
         ),
       );
 
-      expect(find.text('No persistent state file found at $testPath'), findsOneWidget);
+      expect(find.text('No persistent state file. Try starting a release.'), findsOneWidget);
```

Comment:
There are a few more tests in this file that are showing the path.

Reply:
I don't see anymore. Can you send a link?

Reply:
https://github.com/CaseyHillers/cocoon/blob/426578136aa5d34e336a77d92fa53b799fc1e0ef/release_dashboard/test/widgets/conductor_status_test.dart#L104
https://github.com/CaseyHillers/cocoon/blob/426578136aa5d34e336a77d92fa53b799fc1e0ef/release_dashboard/test/widgets/conductor_status_test.dart#L143
https://github.com/CaseyHillers/cocoon/blob/426578136aa5d34e336a77d92fa53b799fc1e0ef/release_dashboard/test/widgets/conductor_status_test.dart#L104

Reply:
Ah I see, weird vscode didn't catch those